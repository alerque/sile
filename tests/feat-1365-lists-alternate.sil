\begin[papersize=a6, class=book]{document}
\use[module=packages.lists]
\font[family=Libertinus Serif]% Default Gentium lacks the white circle.
\nofolios
\neverindent
\set[parameter=document.parskip, value=5pt]%
\set[parameter=lists.parskip, value=2pt]%
% Test case: Nested itemize and enumerate with specified skips, alternate nesting method.

Text flow before.
\begin{enumerate}
  \item{Level 1 Enum
    \begin{itemize}
      \item{Level 2 Item
        \begin{enumerate}
          \item{Level 3 Enum}
        \end{enumerate}}
    \end{itemize}}
\end{enumerate}
Text flow after, just longer to be closer to the last list level.

Text flow before.

\begin{enumerate}
  \item{Level 1 Enum
    \begin{itemize}
      \item{Level 2 Item
        \begin{enumerate}
          \item{Level 3 Enum}
        \end{enumerate}
      Back to level 2 Item}
    \end{itemize}
  Back to level 1 Enum}
\end{enumerate}

Text flow after, just longer to be closer to the last list level.

\end{document}
