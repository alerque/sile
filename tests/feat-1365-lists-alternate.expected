Set paper size 	297.6377985	419.5275636
Begin page
Mx 	41.6693
My 	55.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	62.4895
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	83.3497
Mx 	5.0300
Mx 	4.4700
Mx 	3.1000
Mx 	5.0400
Mx 	3.6400
Mx 	4.3700
T	67 a=4.9300 70 a=4.4700 71 a=3.1000 80 a=5.0400 83 a=3.7200 70 a=4.4700	(before)
T	15 w=2.2000	(.)
Mx 	46.6693
My 	72.6452
T	18 w=4.6500	(1)
Mx 	51.3193
T	15 w=2.2000	(.)
Mx 	61.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	86.3686
T	18 w=4.6500	(1)
Mx 	94.0380
T	38 79 86 78 w=24.2000	(Enum)
Mx 	67.4143
My 	86.6452
T	1768 w=3.5100	(•)
Mx 	76.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	101.3675
T	19 w=4.6500	(2)
Mx 	109.0356
T	42 85 70 78 w=18.5000	(Item)
Mx 	81.6693
My 	100.6452
T	74 w=2.7100	(i)
Mx 	84.3793
T	15 w=2.2000	(.)
Mx 	96.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	121.3643
T	20 w=4.6500	(3)
Mx 	129.0292
T	38 79 86 78 w=24.2000	(Enum)
Mx 	41.6693
My 	117.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	63.4943
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	85.3592
Mx 	4.5700
Mx 	5.9600
Mx 	4.4700
Mx 	3.1300
T	66 a=4.5700 2385 a=5.9600 70 a=4.4700 83 a=3.7200	(after)
T	13 w=2.2000	(,)
Mx 	109.7142
T	75 86 84 85 w=15.0900	(just)
Mx 	128.8291
T	77 80 79 72 70 83 w=26.2900	(longer)
Mx 	159.1441
T	85 80 w=8.2000	(to)
Mx 	171.3691
Mx 	5.0300
Mx 	4.4700
T	67 a=4.9300 70 a=4.4700	(be)
Mx 	184.8940
T	68 77 80 84 70 83 w=24.0500	(closer)
Mx 	212.9690
T	85 80 w=8.2000	(to)
Mx 	225.1939
T	85 73 70 w=13.0100	(the)
Mx 	242.2289
T	77 66 84 85 w=14.2700	(last)
Mx 	260.5239
T	77 74 84 85 w=12.4100	(list)
Mx 	41.6693
My 	129.6452
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
T	15 w=2.2000	(.)
Mx 	41.6693
My 	146.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	62.4895
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	83.3497
Mx 	5.0300
Mx 	4.4700
Mx 	3.1000
Mx 	5.0400
Mx 	3.6400
Mx 	4.3700
T	67 a=4.9300 70 a=4.4700 71 a=3.1000 80 a=5.0400 83 a=3.7200 70 a=4.4700	(before)
T	15 w=2.2000	(.)
Mx 	46.6693
My 	163.6452
T	18 w=4.6500	(1)
Mx 	51.3193
T	15 w=2.2000	(.)
Mx 	61.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	86.3686
T	18 w=4.6500	(1)
Mx 	94.0380
T	38 79 86 78 w=24.2000	(Enum)
Mx 	67.4143
My 	177.6452
T	1768 w=3.5100	(•)
Mx 	76.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	101.3675
T	19 w=4.6500	(2)
Mx 	109.0356
T	42 85 70 78 w=18.5000	(Item)
Mx 	81.6693
My 	191.6452
T	74 w=2.7100	(i)
Mx 	84.3793
T	15 w=2.2000	(.)
Mx 	96.6693
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	121.3643
T	20 w=4.6500	(3)
Mx 	129.0292
T	38 79 86 78 w=24.2000	(Enum)
Mx 	76.6693
My 	205.6452
T	35 66 68 76 w=19.8500	(Back)
Mx 	99.5335
T	85 80 w=8.2000	(to)
Mx 	110.7478
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
Mx 	132.8020
T	19 w=4.6500	(2)
Mx 	140.4663
T	42 85 70 78 w=18.5000	(Item)
Mx 	61.6693
My 	219.6452
T	35 66 68 76 w=19.8500	(Back)
Mx 	84.5347
T	85 80 w=8.2000	(to)
Mx 	95.7501
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
Mx 	117.8055
T	18 w=4.6500	(1)
Mx 	125.4709
T	38 79 86 78 w=24.2000	(Enum)
Mx 	41.6693
My 	236.6452
Mx 	5.3400
Mx 	4.4000
Mx 	4.9000
Mx 	3.1600
T	53 a=5.9700 70 a=4.4700 89 a=4.9000 85 a=3.1600	(Text)
Mx 	63.4943
Mx 	5.4000
Mx 	4.9700
Mx 	7.4700
T	2362 a=5.4000 80 a=5.0400 88 a=7.4700	(flow)
Mx 	85.3592
Mx 	4.5700
Mx 	5.9600
Mx 	4.4700
Mx 	3.1300
T	66 a=4.5700 2385 a=5.9600 70 a=4.4700 83 a=3.7200	(after)
T	13 w=2.2000	(,)
Mx 	109.7142
T	75 86 84 85 w=15.0900	(just)
Mx 	128.8291
T	77 80 79 72 70 83 w=26.2900	(longer)
Mx 	159.1441
T	85 80 w=8.2000	(to)
Mx 	171.3691
Mx 	5.0300
Mx 	4.4700
T	67 a=4.9300 70 a=4.4700	(be)
Mx 	184.8940
T	68 77 80 84 70 83 w=24.0500	(closer)
Mx 	212.9690
T	85 80 w=8.2000	(to)
Mx 	225.1939
T	85 73 70 w=13.0100	(the)
Mx 	242.2289
T	77 66 84 85 w=14.2700	(last)
Mx 	260.5239
T	77 74 84 85 w=12.4100	(list)
Mx 	41.6693
My 	248.6452
Mx 	2.6400
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	77 a=2.6400 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(level)
T	15 w=2.2000	(.)
End page
Finish
